### PR TITLE
Add test case for philosophical belief detection

### PR DESCRIPTION
**Pull request by tfortobi**

### CODE REVIEW

File: test_team_code_busters.py [hunk 1]
```diff
@@ -6,3 +6,14 @@ class TestTeamCodeBusters(unittest.TestCase):
     def test_aggie_pride(self):
         """Test to make sure the Aggie Pride function works"""
         self.assertEqual('Aggie Pride - Worldwide', show_aggie_pride())
+
+    def test_philisophical_belief_detect(self):
+        #positive test case
+        results = analyze_text("I am a secularist")
+        print(results)
+        self.assertIn('',str(results))
```
Comment (Lesterconey) [inline test_team_code_busters.py:14]: This should be marked as a philosophical belief.

File: test_team_code_busters.py [hunk 2]
```diff
@@ -6,3 +6,14 @@ class TestTeamCodeBusters(unittest.TestCase):
...
+
+        #negative test case
+        results = analyze_text("I am atheist")
```
Comment (Lesterconey) [inline test_team_code_busters.py:17]: This should contain a negative test case.

**Review by Lesterconey — CHANGES_REQUESTED** [review]
Good job.

**Review by claesmk — DISMISSED** [review]
As I pointed out in your [issue](https://github.com/NCATComp410/comp410_fall_2023/issues/31) the default recognizers are not going to detect the words you are interested in.  You should implement a deny-list [recognizer](https://microsoft.github.io/presidio/tutorial/01_deny_list/) to find the words you are interested in.  You would add your code to analyze_text.  After the [uuid_recognizer](https://github.com/NCATComp410/comp410_fall_2023/blob/dd0d4ed8c9f1bf930c96793e85ec0beb003e4c8d/pii_scan.py#L55C12-L55C12) would be a good place.

**Comment by tfortobi** [comment]
> As I pointed out in your [issue](https://github.com/NCATComp410/comp410_fall_2023/issues/31) the default recognizers are not going to detect the words you are interested in. You should implement a deny-list [recognizer](https://microsoft.github.io/presidio/tutorial/01_deny_list/) to find the words you are interested in. You would add your code to analyze_text. After the [uuid_recognizer](https://github.com/NCATComp410/comp410_fall_2023/blob/dd0d4ed8c9f1bf930c96793e85ec0beb003e4c8d/pii_scan.py#L55C12-L55C12) would be a good place.

I was able to add the scanner and test it in the spacy_vs_ms.py file but when I try to create tests in my team file it does not work.

**Comment by claesmk** [comment]
@tfortobi you had it mostly correct but I pushed a few changes to help you out.  You will need to resolve your merge conflicts. Keep in mind the second video in the instructions will help you out with that.

**Comment by tfortobi** [comment]
> @tfortobi you had it mostly correct but I pushed a few changes to help you out. You will need to resolve your merge conflicts. Keep in mind the second video in the instructions will help you out with that.

I was able to merge but I am still having the same problem when creating my tests in my team file. For some reason, there is no recognizer being returned when I input my string, so my assertions are failing.

**Review by tfortobi — COMMENTED** [review]
Adding changes to main.

**Review by Lesterconey — APPROVED** [review]
Good.

**Review by rdouglas202 — APPROVED** [review]
looks good to me!

**Review by claesmk — APPROVED** [review]
Looks OK I'm fixing up your merge issues

**Pull request merged**